modules/tfm/zephyr/Kconfig

```diff
@@ -19,4 +19,72 @@ config TFM_SECURE_UART1
 	bool
 	select NRF_UARTE1_SECURE
 
+config TFM_UART0_TXD_PIN
+	int "UART0 TXD pin"
+	default 29 if BOARD_NRF9160DK_NRF9160_NS
+	default 20 if BOARD_NRF5340DK_NRF5340_CPUAPP_NS
+	default 4294967295 # 0xffffffff i.e. disconnected
+	help
+	  The GPIO pin the TFM build system will use for UART0 TXD pin.
+	  UART0 is used by TF-M as the log output for the non-secure image.
+
+	  Note: The non-secure image is only ever used when running the
+	        TF-M regression tests.
```

Comment:
a few of questions:

- Why does TF-M require pin information? How is this used?
- Can't TF-M use pinctrl API to configure pins? If not, why?
- In case pins are mandatory, why not use Devicetree to configure TF-M pins?